googlemock/include/gmock/gmock-generated-matchers.h

```diff
@@ -994,6 +994,211 @@ UnorderedElementsAre(const T1& e1, const T2& e2, const T3& e3, const T4& e4,
       e6, e7, e8, e9, e10));
 }
 
+template <typename T1>
+inline internal::ContainsSequenceMatcher<
+    ::testing::tuple<typename internal::DecayArray<T1>::type> >
+ContainsSequence(const T1& e1) {
```

Comment:
We'd almost immediately need a "ContainsSequenceArray" (bikeshedded name), to take more than 10 matchers, or a dynamic collection of matchers. Add that?  If you had that, you might not need any of these overloads, really.